addons/medical_blood/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="medical_blood">
         <Key ID="STR_ACE_medical_blood_subCategory">
             <English>Bleeding Effect</English>
+            <German>Verblutungseffekt</German>
```

Comment:
"Blut effekt" "Blutungs effekt"
This also happens when you just bleed a little. Doesn't mean you are bleeding out.
"Verbluten" sounds very critical.

Reply:
Changed